tests/testautotls.nim

```diff
@@ -0,0 +1,69 @@
+# Nim-Libp2p
+# Copyright (c) 2023 Status Research & Development GmbH
+# Licensed under either of
+#  * Apache License, version 2.0, ([LICENSE-APACHE](LICENSE-APACHE))
+#  * MIT license ([LICENSE-MIT](LICENSE-MIT))
+# at your option.
+# This file may not be copied, modified, or distributed except according to
+# those terms.
+
+{.push raises: [].}
+
+# import std/[strformat, net] # uncomment after re-enabling AutoTLSManager
+import chronos
+import chronos/apps/http/httpclient
+import
+  ../libp2p/
+    [
+      stream/connection,
+      upgrademngrs/upgrade,
+      autotls/acme/api,
+      autotls/acme/utils,
```

Comment:
`import autotls/acme/utils`
it doesn't really make sense to import internal utils in test...

Reply:
For ACMEError